### PR TITLE
Siren event fixes+conservator update

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
Siren event is now locked behind player amounts. (Requires ~~6~~ 8 crew on to fire)
Also updates conservators, giving them a lethal laser with high agony. Conservators will now construct special barricades and traps. 
So called Sirencades allow siren mobs to shoot through it, but will block all non-beam weapons.
The shock trap conservators deploy electrocutes whoever walks over it and unleashes a flash of light.

## Why It's Good For The Game
Makes conservators less of a joke. Fixes lowpop Siren calls.

## Changelog
```changelog
add: Sirencades
add: Siren Shocktraps
tweak: Sirenscan events are now rarer, and should no longer spawn on players ever. 
Tweak:Siren spawn pods now spawn together, meaning beware deadly synergy
tweak:conservators now have a limit of 5 structures they canbuild
tweak: conservator AI behavior
remove: Code relating to siren mobs attacking hidden targets. Kept runtiming.
fix:siren shields now actually block bullets and no longer take damage while doing so.
tweak: adjusted siren spawns so that most contain atleast one conservator.
balance: Conservators now are more dangerious, and have more 'utility'
balance:augmentors now do less damage per shot, and projectiles are a bit slower.
imageadd: ported TG sprites for barricade/trap
imageadd: 3 new sprites for 3 new siren mobs

```

### CODE REVIEW

File: zzzz_modular_occulus/code/game/gamemodes/events/siren_scan.dm [hunk 1]
```diff
@@ -9,41 +9,41 @@
 
 	event_type = /datum/event/siren_scan
 	event_pools = list(EVENT_LEVEL_MODERATE = POOL_THRESHOLD_MODERATE)
```
Comment (Mocha-Not-Latte) [inline zzzz_modular_occulus/code/game/gamemodes/events/siren_scan.dm:11]: Please make this Major instead

Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/game/gamemodes/events/siren_scan.dm:11]: I think it's fine as is.

**Comment by Jamini** [comment]
You are failing CI

**Comment by RadiantFlash7** [comment]
> You are failing CI

I believe I don’t have whatever fix you added to make the CI issue to stop screaming. I can update the PR when I’m home. It would likely fix the issue

**Comment by RadiantFlash7** [comment]
Right uh. Fixed my compile issue with Windows build and compile maps. I don't understand whats wrong CI Linters and Integrations.
Any help on that is appreciated

**Comment by bearrrrrrrr** [comment]
Why is this set to moderate?

**Comment by bearrrrrrrr** [comment]
ignore muh deleted comment, after looking stuff over moderate seems...fine, in terms of danger? i still think it thematically fits in major but eh w/e

**Comment by RadiantFlash7** [comment]
> ignore muh deleted comment, after looking stuff over moderate seems...fine, in terms of danger? i still think it thematically fits in major but eh w/e

Theres plans to add a miniboss mob in an event that will be a major version of the sirenscan. As is now, I'm hoping to keep this under moderate- I need to live test the new spawn system to be certain. Grouping the mobs together in the way it is may have made them too tough for moderate.

**Review by Jamini — APPROVED** [review]

**Pull request merged**